### PR TITLE
Feature/layout update

### PR DESCRIPTION
**Pull request by bcorbold**
- Front end now connects to the backend and is completely driven from it's responses!
- Restyled application to a more "flat" style

User View
screen shot 2018-01-27 at 10 46 07 pm 2

Bartender View
screen shot 2018-01-27 at 10 47 03 pm 2

### CODE REVIEW

**Comment by bcorbold** [comment]
Some of the styling is still really rough... Going to do what I can, but can also be finished after rev0 presentation

File: src/app/components/views/login/login.component.ts [hunk 1]
```diff
@@ -17,7 +17,10 @@ export class LoginComponent {
 
   constructor(private router: Router, private accountService: AccountService) {}
 
-  attemptLogin() {
+  attemptLogin(): void {
+    // this.username = 'admin';
```
Comment (JesseTatasciore) [inline src/app/components/views/login/login.component.ts]: delete

File: src/app/services/message/message.service.ts [hunk 2]
```diff
@@ -1,278 +1,311 @@
+/* tslint:disable */
```
Comment (JesseTatasciore) [inline src/app/services/message/message.service.ts]: Matt would be disapointed

File: src/app/services/message/message.service.ts [hunk 3]
```diff
@@ -1,278 +1,311 @@
...
-  }
-];
-
 @Injectable()
 export class MessageService implements OnDestroy {
 
-  private pollingSubscription: Subscription;
+  private static BATCH_POLLING_RATE = 5000;
+  private static ORDER_POLLING_RATE = 5000;
+  private static WAITING = 'Wiating';
```
Comment (JesseTatasciore) [inline src/app/services/message/message.service.ts]: Waiting

**Pull request merged**